sdk/src/utils/mod.rs

```diff
@@ -17,6 +17,7 @@ pub(crate) mod hash_utils;
 #[allow(dead_code)] // for wasm build
 pub(crate) mod patch;
 #[cfg(feature = "file_io")]
```

Comment:
I think there's a `cfg(all(feature(...)` syntax that might be shorter here.

Reply:
done